asn1ate/asn2quickder.py

```diff
@@ -0,0 +1,321 @@
+#!/usr/bin/python
+#
+# asn2quickder -- Generate header files for C for use with Quick `n' Easy DER
+#
+# This program owes a lot to asn1ate, which was built to generate pyasn1
```

Comment:
Thank you! :-)

But this is not necessary here, better to add a license header and a copyright statement. I can update LICENSE.txt to mention you as owner of the asn2quickder module, or if you can add something as part of your PR.